meta_test.go

```diff
@@ -94,3 +95,54 @@ func TestEncodeDecode(t *testing.T) {
 		t.Fatalf("decoded object not equal:\nexpected %#v\nactual   %#v", v, w)
 	}
 }
+
+func benchmarkEncode(n int, t *testing.B) {
+	type test struct {
+		Array []string `json:"array"`
+	}
+	v := &test{}
+	for j := 0; j < n; j++ {
+		v.Array = append(v.Array, "t")
+	}
+	t.ReportAllocs()
+	for i := 0; i < t.N; i++ {
```

Comment:
Reset the timer with [`B.ResetTimer`](https://golang.org/pkg/testing/#B.ResetTimer).

Reply:
Add ResetTimer at 51c00f3 though it does not have any effect on the result.